cmd/dlt/machinepool/nodepool.go

```diff
@@ -0,0 +1,43 @@
+package machinepool
+
+import (
+	"os"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/openshift/rosa/pkg/interactive/confirm"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+func deleteNodePool(r *rosa.Runtime, nodePoolID string, clusterKey string, cluster *cmv1.Cluster) {
+	// Try to find the machine pool:
+	r.Reporter.Debugf("Loading nodepools for hosted cluster '%s'", clusterKey)
+	nodePools, err := r.OCMClient.GetNodePools(cluster.ID())
+	if err != nil {
+		r.Reporter.Errorf("Failed to get nodepools for cluster '%s': %v", clusterKey, err)
```

Comment:
Should this (and other errors across the various commands) be `machine pools` instead of `nodepools`, considering they will be displayed to the user?

Reply:
yes, indeed, I saw I mixed the terminology.
I think I'm going to use "machine pools for hosted clusters" when referring to nodepools.
Then things will be more consistent across all debug/error/info messages.